polaris-for-vscode/src/server.ts

```diff
@@ -69,6 +69,7 @@ type GroupedCompletionItemPatterns = {
 
 const groupedCompletionItemPatterns: GroupedCompletionItemPatterns = {
   breakpoints: /width/,
+  border: /border/,
```

Comment:
Unsure if this is a conflict with line `81` https://github.com/Shopify/polaris/pull/8906/files#diff-08ed22c5803ae3fd2e9895f8e863b69c24e2e15891e60421963ba8ede292a6bdR81

Reply:
I ended up removing the `shape` autocomplete as we want people to use the border tokens anyway.